presto-spi/src/main/java/io/prestosql/spi/connector/ConnectorMetadata.java

```diff
@@ -232,6 +232,14 @@ default void renameTable(ConnectorSession session, ConnectorTableHandle tableHan
         throw new PrestoException(NOT_SUPPORTED, "This connector does not support renaming tables");
     }
 
+    /**
+     * Comments to the specified table
+     */
+    default void commentTable(ConnectorSession session, ConnectorTableHandle tableHandle, String comment)
```

Comment:
Let's name this `setTableComment` and also change `String` to be `Optional<String>`. This aligns with how comment is represented in `ConnectorTableMetadata` and allows a single call to add/update/remove.